clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -241,7 +240,15 @@ public FSDataOutputStream append(Path path, int i, Progressable progressable) th
     /**
      * This method is implemented under the following assumptions:
```

Comment:
```suggestion
     * Rename, behaving similarly to the POSIX "mv" command, but non-atomically.
```
(assuming that is true...).